noir-projects/aztec-nr/aztec/src/encrypted_logs/encrypted_note_emission.nr

```diff
@@ -23,15 +23,8 @@ fn compute_raw_note_log<Note, let N: u32, let M: u32>(
 
     let contract_address: AztecAddress = context.this_address();
 
-    let encrypted_log: [u8; M] = compute_encrypted_note_log(
-        contract_address,
-        storage_slot,
-        ovsk_app,
-        ovpk,
-        ivpk,
-        recipient,
-        note
-    );
+    let plaintext = note.to_be_bytes(storage_slot);
```

Comment:
Actually, `to_be_bytes` in general needs a renaming. Thoroughly confused as to why this might receive a storage slot.